src/routes/index.svelte

```diff
@@ -26,32 +27,38 @@
 		</p>
 	</div>
 
-	<div class="flex justify-around h-96">
+	<div>
 		<img
-			class="object-scale-down h-96 w-120 border-2 border-header-blue"
-			src="house.jpeg"
-			alt="Front view of the house"
-		/>
-		<img
-			class="object-scale-down h-96 w-120 border-2 border-header-blue"
-			src="rustic.jpeg"
-			alt="Rustic accommodation view"
+			class="object-scale-down"
+			src="dealhp.png"
+			alt="Aerial view of Deal seafront"
 		/>
 	</div>
 
-	<div class="flex justify-around h-96 py-20">
-		<a href="/gallery" class="relative h-96">
-			<div
-				class="absolute w-full h-1/3 bg-gradient-to-l from-gray-50 to-transparent right-0 mt-0.5 mr-0.5 opacity-80"
-			>
-				<h3 class="absolute right-0">Gallery</h3>
+	<div class="flex flex-row justify-center space-x-14">
+		<a href="/about-tss">
+			<div class="w-52 h-28 mt-10 p-2 flex items-center text-center bg-light-blue border-4 border-header-blue hover:border-slate-800 hover:bg-mid-blue">	
```

Comment:
If this is the same styling used across the 3 buttons let's create a new CSS class to save having to copy and paste this. 

This will go on line 70 and you can call it something unique, e.g. `quick-links`